app/controllers/bakeries_controller.rb

```diff
@@ -0,0 +1,25 @@
+class BakeriesController < ApplicationController
+
+  def index
+    @bakeries = Bakery.all
+  end
+
+  def show
+    @bakery = Bakery.find(params[:id])
+  end
+
+  def new
+    @bakery = Bakery.new
+  end
+
+  def create
+    @bakery = Bakery.find(params_bakery)
```

Comment:
@bakery = Bakery.new(params_bakery)
@bakery.user = current_user
if @bakery.save
 redirect_to bakery_path(@bakery)
else
@baker.new
 #render the propor page
end